### PR TITLE
feat(design-system): add xds filters

### PR DESCRIPTION
**Pull request by diegopf**
EMP-120

### CODE REVIEW

File: src/components/search/facets/custom-sliced-filters.vue [hunk 1]
```diff
@@ -1,13 +1,17 @@
 <template>
   <SlicedFilters v-bind="$attrs">
     <template #show-more>
-      <div class="x-list x-list--horizontal x-list--align-center x-list--gap-03">
+      <div
```
Comment (ajperezbau) [inline src/components/search/facets/custom-sliced-filters.vue:4]: I think we can remove this `div` and just place here the icon and the text.

File: src/components/search/facets/custom-sliced-filters.vue [hunk 2]
```diff
@@ -1,13 +1,17 @@
 <template>
   <SlicedFilters v-bind="$attrs">
```
Comment (ajperezbau) [inline src/components/search/facets/custom-sliced-filters.vue:2]: I think we should add a dynamic prop to add classes to the `sliced-filters` buttons.
After that, we would just apply the `x-facet-filter-lg` class here.

File: src/components/search/facets/custom-facets.vue
Comment (ajperezbau) [inline src/components/search/facets/custom-facets.vue:1]: Should we use the rounded badge or the default one in the selected filter count?

image

Comment (diegopf) [reply src/components/search/facets/custom-facets.vue:1]: I don't think we have to change anything there, at least for now.

**Review by mnavarroespinosa — DISMISSED** [review]
good job!

**Review by ajperezbau — APPROVED** [review]
Good job 👌

**Review by mnavarroespinosa — APPROVED** [review]

**Pull request merged**